package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "setup:docs": "./scripts/download-dependencies.sh",
     "setup:native": "./scripts/download-dependencies.sh && ./scripts/setup-react-android.sh",
-    "postinstall": "yarn-deduplicate && expo-yarn-workspaces check-workspace-dependencies && yarn workspace @expo/cli prepare",
+    "postinstall": "yarn-deduplicate && expo-yarn-workspaces check-workspace-dependencies && yarn workspace @expo/cli prepare && patch-package",
```

Comment:
nit * 3:
personally if i only have just one patch and that's temporary, i would prefer not to add patch-package and introduce extra dependency to the monorepo.
something like the reverse of #19516 was what i did before.